test/test_sentry_processor.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+import pytest
+
+from structlog_sentry import SentryProcessor
+
+
+def test_sentry_disabled():
+    processor = SentryProcessor(active=False)
+    event_dict = processor(None, None, {"level": "error"})
+    assert event_dict.get("sentry") != "sent"
+
+
+def test_sentry_skip():
+    processor = SentryProcessor()
+    event_dict = processor(None, None, {"sentry_skip": True, "level": "error"})
+    assert event_dict.get("sentry") != "sent"
+
+
+def test_sentry_sent():
+    processor = SentryProcessor()
+    event_dict = processor(None, None, {"level": "error"})
+    assert event_dict.get("sentry") == "sent"
+
+
+@pytest.mark.parametrize("level", ["debug", "info", "warning"])
+def test_sentry_log(mocker, level):
+    m_capture_event = mocker.patch("structlog_sentry.capture_event")
+
+    event_data = {"level": level, "event": level + " message"}
+    processor = SentryProcessor(sentry_log_level=logging.getLevelName(level.upper()))
+    processor(None, None, event_data)
+
+    m_capture_event.assert_called_once_with(
+        {
+            "level": level,
+            "message": event_data["event"],
+            "extra": event_data,
+        },
+        hint=None,
+    )
+
+    processor_only_errors = SentryProcessor(sentry_log_level=logging.ERROR)
+    event_dict = processor_only_errors(
+        None, None, {"level": level, "event": level + " message"}
+    )
+
+    assert event_dict.get("sentry") != "sent"
+
+
+@pytest.mark.parametrize("level", ["error", "critical"])
+def test_sentry_log_failure(mocker, level):
+    m_capture_event = mocker.patch("structlog_sentry.capture_event")
+    mocker.patch(
+        "structlog_sentry.sentry_utils.event_from_exception",
+        return_value=({"exception": "exception"}, "hint"),
```

Comment:
As values use Mock [Sentinel](https://docs.python.org/3/library/unittest.mock.html#sentinel) objects instead of arbitrary strings. 
```python
return_value=({"exception": mocker.sentinel.exception}, mocker.sentinel.hint)
```

Reply:
The reason is that when you are asserting them later in test, you can be sure that your patch worked and values has been propagated trough code. It prevent's bugs.